Doc/faq/windows.rst

```diff
@@ -170,11 +170,13 @@ offender.
 How do I make an executable from a Python script?
 -------------------------------------------------
 
-See http://cx-freeze.sourceforge.net/ for a distutils extension that allows you
-to create console and GUI executables from Python code.
-`py2exe <http://www.py2exe.org/>`_, the most popular extension for building
-Python 2.x-based executables, does not yet support Python 3 but a version that
-does is in development.
+See http://cx-freeze.sourceforge.net/ for the distutils extension and
+http://www.pyinstaller.org/ for another package,
+that both allow you to create console and GUI executables
+from Python code.
+`py2exe <http://www.py2exe.org/>`_, popular extension for building
+Python 2.x-based executables, partially supports Python 3, see
+https://pypi.python.org/pypi/py2exe/.
```

Comment:
All three of these do the same or similar things but the grouping of the sentences reads oddly. Can we just simplify this to be more of a list? Something like...

```
The following packages can help with the creation of cross-platform console and GUI executables:
* http://cx-freeze.sourceforge.net/
* http://www.pyinstaller.org/

Additionally, http://www.py2exe.org/ exclusively supports Windows executables.
```